SonarQube.TeamBuild.PreProcessor/ArgumentProcessor.cs

```diff
@@ -347,7 +181,7 @@ private static bool TryGetAnalysisSettings(IEnumerable<ArgumentInstance> argumen
                     AnalysisSetting existing = TryGetAnalysisSetting(setting.Id, settings);
                     if (existing != null)
                     {
-                        logger.LogError(Resources.ERROR_CmdLine_DuplicateArg, argument.Value, existing.Value);
+                        logger.LogError(Resources.ERROR_CmdLine_DuplicateSetting, argument.Value, existing.Value);
```

Comment:
I separated out the resources required by the common component. "ERROR_CmdLine_DuplicateArg" was also used by the pre-processor -specific code to report duplicate dynamic settings, so I copied this resource string and changed the name and message to be more specific.